parsers/IN_GJ.py

```diff
@@ -45,6 +46,19 @@ def to_float(s):
     return float(ast.literal_eval(s))
 
 
+def split_and_sum(expression):
+    """
+    Avoid using literal_eval for simple addition expressions.
+    Returns sum of all positive numbers.
+    """
+
+    split_vals = expression.split('+')
+    float_vals = [float(v) for v in split_vals]
+    total = sum([v for v in float_vals if v > 0.0])
```

Comment:
why keep only positive numbers?

Reply:
can you not replace all `to_float` by `split_and_sum`?

Reply:
@maxbellec the inactive units have a bit of self-consumption, which causes the negative values in the chart. One of them had like -30 MW yesterday, which would reduce the output of coal by the same amount, and therefore total emissions, too, which would be slightly inaccurate.

Reply:
The negative numbers are not correct and including them in the sum reduces the generation for that fuel type. I can replace to_float entirely if you're happy with that.

Reply:
alright! Yeah, replace all `to_float` by `split_and_sum` seems better, I was not a big fan of `literal_eval` either